package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fandom/jwplayer-fandom",
-	"version": "2.7.0",
```

Comment:
I think we should not publish new version without ticket number at the end until the ticket is tested. We should publish just before merging to master

Reply:
Alright, we can do that